src/redux/user/user.types.js

```diff
@@ -0,0 +1,5 @@
+export const SET_USER = 'SET_USER';
+export const LOGIN_USER = 'LOGIN_USER';
+export const SET_ERROR = 'SET_ERROR';
+export const LOGOUT_USER = 'LOGOUT_USER';
+export const USER_LOADING = 'USER_LOADING';
```

Comment:
Please, rename action type to `SET_USER_LOADING`, action types should reflect some action

Reply:
Rename